### PR TITLE
Add basic In Video Quiz functionality

### PR DESCRIPTION
**Pull request by caesar2164**
- Leverages default video and problem components to take advantage of grading and theming
- Basic "in-video" learner view where components are displayed inside of video at the set time
- Instructor view where problems are displayed as normal problems with a message about when they appear in the video so instructors can still use all the problem data features
- Simple Studio view in which the video and problem component IDs can be set, as well as the timemap

@stv

### CODE REVIEW

**Comment by caesar2164** [comment]
I'd like to split the tox and requirements stuff out into a separate commit

**Comment by stvstnfrd** [comment]
I just enabled travis-ci and coveralls for this repo; they should kick in following your next commit.

File: invideoquiz/invideoquiz.py [hunk 1]
```diff
@@ -1,43 +1,86 @@
...
+    video_id = String(
+        display_name=_('Video ID'),
+        default=_(''),
+        scope=Scope.settings,
+        help=_('This is the in video quiz video ID'),
+    )
+
+    timemap = String(
+        display_name=_('Problem Timemap'),
+        default=_(''),
```
Comment (stvstnfrd) [inline invideoquiz/invideoquiz.py]: orly

i18n'ed empty string?
;)

File: tox.ini [hunk 2]
```diff
@@ -12,24 +12,32 @@ deps =
     edx-opaque-keys
     mock
     django_nose>=1.4
+    xblockutils
```
Comment (stvstnfrd) [inline tox.ini:15]: I don't think I knew about this back then;
we should be able to pull this directly from `requirements.txt` [1].

[1] https://testrun.org/tox/latest/example/basic.html#depending-on-requirements-txt

Comment (stvstnfrd) [inline tox.ini:15]: Ah, we're installing directly from `setup.py`; cool.
Can we add most of these deps there?
Then we can just add the "dynamic" ones here:

``` ini
deps =
    -rrequirements.txt
    django >= 1.8, < 1.9
```

**Comment by stvstnfrd** [comment]
We should set this to AGPLv3:
https://github.com/Stanford-Online/xblock-in-video-quiz/pull/1/files#diff-2eeaed663bd0d25b7e608891384b7298R45

**Comment by stvstnfrd** [comment]
Actually, can we include the entire project in the PR, instead of just the changes after cookiecutter (is there an initial empty commit)?

As is, it's not obvious that everything that needs done post-cookiecutter has been addressed (remember, Azim didn't polish that project). There are `TODO` comments (from the template, sure, but we shouldn't merge/release like that), no license is specified in `setup.py`, etc.

Especially since it's already _nearly_ the whole project, it's tough reading with a few snippets missing.

**Comment by caesar2164** [comment]
new PR: https://github.com/Stanford-Online/xblock-in-video-quiz/pull/2

**Pull request closed without merging**